app/models/service/aggregation.rb

```diff
@@ -315,7 +315,7 @@ def aggregate_all_vm_disk_space_used
   def aggregate_all_vm_memory_on_disk
     if has_attribute?("aggregate_all_vm_memory_on_disk")
       # Avoids (poorly) "ERROR:  integer out of range" in postgres
-      self["aggregate_all_vm_memory_on_disk"] * 1.megabyte
+      self["aggregate_all_vm_memory_on_disk"].try(:*, 1.megabyte)
```

Comment:
@bdunne - the textual summaries for a service VM were expecting a non-nil value - I updated those to handle a nil for these aggregate attributes - but found that the nil here for this particular attribute was also not handled. As @NickLaMuro mentions above - these if/else statements should be re-written - but not in the scope of this PR.